assets/javascripts/discourse/helpers/activity-pub-actor-image.js

```diff
@@ -0,0 +1,23 @@
+import { htmlSafe } from "@ember/template";
+import { translateSize } from "discourse-common/lib/avatar-utils";
+import { registerRawHelper } from "discourse-common/lib/helpers";
+import { buildHandle } from "../lib/activity-pub-utilities";
+
+function renderActivityPubActorImage(actor, opts) {
+  opts = opts || {};
+
+  if (actor) {
+    const size = translateSize(opts.size);
+    const url = actor.icon_url || "/images/avatar.png";
+    const title = buildHandle({ actor });
+    return `<img loading='lazy' alt='' width='${size}' height='${size}' src='${url}' title='${title}'>`;
+  } else {
+    return "";
+  }
```

Comment:
Same here, if we can do this without a helper, it would be best.

Reply:
Done: https://github.com/angusmcleod/discourse-activity-pub/commit/3d329999a75948030095e28f10ea5233161cbe33